Rules/mtbmap.mrules

```diff
@@ -1749,17 +1750,15 @@ rules
 				line-color : blue
 				line-width : 7:0.1;13:0.5
 				line-opacity : 0.5
+				text-func : NodeName(e)
 			for : intermittent
 				define
-					fill-color : white
+					fill-color : #B8DBF7
 					fill-hatch : swne
-					fill-hatch-color : #B8DBF7
-					fill-hatch-opacity : 1
+					fill-hatch-color : white
+					fill-hatch-opacity : 0
+					painting-cycle : 403
 			draw : fill
```

Comment:
The "draw : fill" for "area water" is creating duplicate fill, as there is a "draw : fill" for all of "area *" later (at line 1835)